modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml

```diff
@@ -548,6 +559,50 @@ Resources:
                       "log_stream_name": "default_logs_{instance_id}"
                     }
                   }
+            'c:\cfn\scripts\AmazonCloudWatchAgent\cloudwatch-applog-config.py':
+              content: !Sub 
+                - |-
+                  import json
+                  import os
+
+                  print('running cwl baseline modification script.')
```

Comment:
nit: this print line is not present in the lx template

Reply:
removed from the win templates